### ISSUE TITLE
[TMC 2209] Wrong microstep resolution configurations in schematics

### ISSUE THREAD

**Issue by njroussel** [makerbase-mks/MKS-StepStick-Driver#12]
(I believe that I've checked every reasonable resource where this might have already been documented, I'm sorry if this is a well known discrepancy and that I simply missed it so far.)

The TMC 2209 MKS driver schematics [file in this repository](https://github.com/makerbase-mks/MKS-StepStick-Driver/blob/master/MKS%20TMC2209/MKS%20TMC2209%20V2.0_001/MKS%20TMC2209%20V2.0_001%20SCH.pdf) shows the following:
```
+-----+-----+-------+
| MS1 | MS2 | Steps |
+-----+-----+-------+
| GND | GND |     8 |
| VIO | GND |     2 |
| GND | VIO |     4 |
| VIO | VIO |    16 |
+-----+-----+-------+
```

This agrees with the spec sheet for the [TMC 2209 driver spec sheet](https://github.com/makerbase-mks/MKS-StepStick-Driver/blob/master/MKS%20TMC2209/U0000000291%20IC%20TMC2209-LA%20%20QFN28.pdf) that can also be found in this repository.

However, that spec sheet is outdated. The [latest revision](https://www.analog.com/media/en/technical-documentation/data-sheets/TMC2209_datasheet_rev1.09.pdf) (v1.09) shows a different table (section 3.4):
```
+-----+-----+-------+
| MS1 | MS2 | Steps |
+-----+-----+-------+
| GND | GND |     8 |
| VIO | GND |    32 |
| GND | VIO |    64 |
| VIO | VIO |    16 |
+-----+-----+-------+
```
In fact, the revision history (section 25) mentions the following: "V1.01 2019-Apr-26 BD Corrected table of microstep pin settings".

I therefore believe that the schematics files that can be found in this repository are incorrect.

TL;DR: 1/32 and 1/64 microstep resolution (before interpolation) is possible with the MKS TMC 2209 driver and is not limited to 1/16 as currently described in the schematics file in this repository.
